examples/redis_example/requirements.txt

```diff
@@ -0,0 +1,27 @@
+amqp==5.1.1
+async-timeout==4.0.2
+billiard==3.6.4.0
+celery==5.2.7
+certifi==2022.12.7
+charset-normalizer==3.1.0
+click==8.1.3
+click-didyoumean==0.3.0
+click-plugins==1.1.1
+click-repl==0.2.0
+gevent==22.10.2
+greenlet==2.0.2
+idna==3.4
+kombu==5.2.4
+prompt-toolkit==3.0.36
+pytz==2022.7.1
+redis==4.5.1
+requests==2.28.2
+rq==1.13.0
+six==1.16.0
+snowplow-tracker @ file:///Users/jackkeene/Documents/Work/snowplow-python-tracker
```

Comment:
These are too many requirements and not all of them are necessary. I think only `gevent` and `redis` are necessary, this is also what the [`setup.py`](https://github.com/snowplow/snowplow-python-tracker/blob/master/setup.py#L72-L74).

Also the `snowplow-tracker` has a path for your local computer. I think we can skip that one and instead add a step in the README to run `setup.py` (I'll comment on that in readme).

Reply:
Seems like this file was removed,  I didn't mean to suggest that, it was useful for installing the `gevent` and `redis` dependencies. But I think it should only contain those two and nothing else. So I would set it like:

```
redis~=4.5
gevent~=22.10
```